### DISCUSSION TITLE
More filters on tag fields

### DISCUSSION THREAD

**Discussion by pierre-H** [directus/directus#20122] [Feature Requests]
### Summary

Hello,

It would be great to have for `tags` fields a filter `contains` and `doesn't contains`.

### Motivation

For insights or search

### Detailed Design

I don't know

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

According to the DB platform it might be complex to create a filter on JSON field

### Alternatives

None

### Adoption Strategy

very easy to use

**Comment by rijkvanzanten** [comment]
This is a duplicate of https://github.com/directus/directus/discussions/7277 🙂
